setup.py

```diff
@@ -94,8 +94,5 @@ def get_version(*file_paths):
         "lms.djangoapp": [
             'eox_nelp = eox_nelp.apps:EoxNelpConfig',
         ],
-        "cms.djangoapp": [
```

Comment:
why did you remove this ?

Reply:
Studio doesn't have mfe config app  and eox-theming configurations.

![2023-02-06_17-23](https://user-images.githubusercontent.com/51926076/217101427-ffbcc8c7-a1da-4d0c-a5c9-b746d4c71798.png)
![2023-02-06_17-20](https://user-images.githubusercontent.com/51926076/217101433-33ff0fa2-e788-4940-bc90-84be370dd086.png)
So this plugin should not work in studio. or what another approach could we have? CHanging the entry URLs?

Reply:
solved in [18130d3](https://github.com/eduNEXT/eox-nelp/pull/17/commits/18130d3eb0b784ac3ea29e52a45d20f401d7ac64) and [e2cd6d5](https://github.com/eduNEXT/eox-nelp/pull/17/commits/e2cd6d5ce0d3d1a6d8169f874fcaeb2f8a2a28b5)